### PR TITLE
Update asnct.is-a.dev

### PR DESCRIPTION
**Pull request by asnct**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/asnct.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+  "description": "Redirect to my telegram channel & mail",
   "owner": {
     "username": "asnct",
     "email": "asnct@proton.me"
   },
   "record": {
     "URL": "https://asnct_bio.t.me",
-    "TXT": "mailru-domain: wDSaV835X4LZyKsP"
+    "TXT": "mailru-domain: wDSaV835X4LZyKsP",
+    "MX": "emx.mail.ru."
```
Comment (Stef-00012) [inline domains/asnct.json:10]: remove the trailing dot

**Review by Stef-DP — DISMISSED** [review]

File: domains/asnct.json (hunk 1 above)
Comment (asnct) [reply domains/asnct.json]: > remove the trailing dot

Done

File: domains/asnct.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+  "description": "Redirect to my telegram channel & mail",
   "owner": {
     "username": "asnct",
     "email": "asnct@proton.me"
   },
   "record": {
     "URL": "https://asnct_bio.t.me",
-    "TXT": "mailru-domain: wDSaV835X4LZyKsP"
+    "TXT": "mailru-domain: wDSaV835X4LZyKsP",
+    "TXT": "v=spf1 redirect=_spf.mail.ru",
```
Comment (DEV-DIBSTER) [inline domains/asnct.json:10]: When you have multiple values for the same record, use an array.

```json

{
   "TXT": ["mailru-domain: wDSaV835X4LZyKsP", "v=spf1 redirect=_spf.mail.ru"]
}

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/asnct.json (hunk 2 above)
Comment (asnct) [reply domains/asnct.json]: Done

**Pull request merged**